crates/swc_css_codegen/tests/fixture/at-rules/keyframes/3/output.min.css

```diff
@@ -0,0 +1 @@
+@keyframes hahaha{0%,1%{color:red}100%{color:red}}
```

Comment:
`100%` can be `to`

Reply:
Yes, we don't support minimizing them currently, it is just codegen (i.e. not AST transformations), but yes, we will improve it in near future